views/index.jade

```diff
@@ -40,6 +39,7 @@ block content
             option(value='pyramid') pyramid
             option(value='bizstyle') bizstyle
             option(disabled='') PyPi Custom Themes
+            option(value='sphinx_fossasia_theme') sphinx_fossasia_theme
```

Comment:
I suppose so. But that section will only contain a single theme right? would that look ok?